content/dart/concepts/type-conversion/type-conversion.md

```diff
@@ -0,0 +1,496 @@
+
+## Example toString Output 1
+
+```dart
+void main() {
+  var doubleValue1 = 2000000.2/2;
+  var doubleValue2 = 1/2000000;
+
+  var strValue1 = doubleValue1.toString();
+  var strValue2 = doubleValue2.toString();
+
+  print(strValue1);
+  print(strValue2);
+}
+```
+
+The `Shell` output:
+
+```Shell
+1000000.1
+5e-7
+```
+
+For special values, returns `NaN` if the input is `NaN`, returns `Infinity`, if input is `Infinity`, and `-Infinity`, if the input is `-Infinity`.
+
+## Example toString Output 2
+
+```dart
+void main() {
+  var doubleValue1 = 0.0/0.0;
+  var doubleValue2 = 1.0/0.0;
+  var doubleValue3 = -1.0/0.0;
+
+  var strValue1 = doubleValue1.toString();
+  var strValue2 = doubleValue2.toString();
+  var strValue3 = doubleValue3.toString();
+
+  print(strValue1);
+  print(strValue2);
+  print(strValue3);
+}
+```
+
+The `Shell` output:
+
+```Shell
+NaN
+Infinity
+-Infinity
+```
+
+An `int` is converted to a decimal representation with no decimal point.
+
+## Example toString Output 3
+
+```dart
+void main() {
+  var intValue1 = 123e18;
+  var intValue2 = 123e20;
+
+  var strValue1 = intValue1.toString();
+  var strValue2 = intValue2.toString();
+
+  print(strValue1);
+  print(strValue2);
+}
+```
+
+The `Shell` output:
+
+```Shell
+123000000000000000000
+1.23e+22
+```
+
+---
+
+## Variations Of `toString` Method
+
+* `toStringAsFixed()`[^6]
+* `toStringAsExponential()`[^7]
+* `toStringAsPrecision()`[^8]
+
+---
+
+## toStringAsFixed() Method
+
+## Implementation
+
+```pseudo
+String toStringAsFixed(int fractionDigits);
+```
+
+The `toStringAsFixed` method converts the input number to a `double` before computing the string representation of that number.
+
+The parameter `fractionDigits` must be an integer satisfying:
+
+  0 <= `fractionDigits` <= 20.
+
+Meaning the `fractionDigits` must be a Non-Zero and a Positive value and must be less than 21.
+
+If `fractionDigits` equals 0, then the decimal point is omitted.
+
+If the absolute value of the input is greater than or equal to 10^21, then this methods returns an exponential representation computed by `toStringAsExponential()`.
+
+## toStringAsFixed Method Example
+
+```dart
+  void main() {
+  var a = 1.toStringAsFixed(3);
+  var b = (256463.51554546).toStringAsFixed(3);
+  var c = (2351.44464661).toStringAsFixed(5);
+  var d = 254645466111.toStringAsFixed(3);
+  var e = 1224.toStringAsFixed(20);
+  var f = 5.25.toStringAsFixed(0);
+  
+  print(a);
+  print(b);
+  print(c);
+  print(d);
+  print(e);
+  print(f);
+}
+```
+
+The `Shell` output:
+
+```Shell
+1.000
+256463.516
+2351.44465
+254645466111.000
+1224.00000000000000000000
+5
+```
+
+> **Note:** If the `fractionDigit` value is not in specified range, console output will display a `RangeError`.
+
```

Comment:
We don't need any specific function explanation or example in this case. Please remove the above-mentioned lines.